app/models/miq_queue.rb

```diff
@@ -47,7 +47,10 @@ def self.messaging_client(client_ref)
       # caching the client works, even if the connection becomes unavailable
       # internally the client will track the state of the connection and re-open it,
       # once it's available again - at least thats true for a stomp connection
-      ManageIQ::Messaging::Client.open(messaging_client_options.merge(:client_ref => client_ref))
+      options = messaging_client_options&.merge(:client_ref => client_ref)
+      return if options.nil?
```

Comment:
If you move this line up, then you don't need the `&.merge`

Reply:
Should we also log an error message?  Something along the lines of `"Messaging type set to #{messaging_type}, but no configuration provided, falling back to miq_queue"`

Reply:
Without some rate limiting that would be a log message for every event and metric

Reply:
I think when we are depending on the Messaging::Client connection we should raise an exception if the config is missing, right now it is just "in addition to" so wanted to keep the noise to a minimum

Reply:
ok